src/SmartComponents/Inventory/applications/advisor/Advisor.js

```diff
@@ -82,17 +93,21 @@ class InventoryRuleList extends Component {
     }
 
     buildRuleCards = () => {
-        const { activeReports, expanded, kbaDetails } = this.state;
+        const { activeReports, expanded, kbaDetails, remediation } = this.state;
 
         return (
             <Fragment>
-                <div className="pf-u-display-flex pf-u-flex-row-reverse">
+                <div className="pf-u-display-flex pf-u-flex-row-reverse ins-c-advisor-system-actions">
```

Comment:
Can we just use the `<Level>` component here instead?

Reply:
@ryelo done